fastlane/Fastfile

```diff
@@ -735,49 +737,72 @@ platform :android do
 
   def build_android
     config_mode = ENV['BUILD_FOR_RELEASE'] == 'true' ? 'Release' : 'Debug'
+    build_task = ENV['ANDROID_BUILD_TASK'].nil? || ENV['ANDROID_BUILD_TASK'].empty? ? 'assemble' : ENV['ANDROID_BUILD_TASK']
+    build_task = build_task.split(",")
+    if build_task.include?("assemble")
```

Comment:
shouldn't this be `build_task == 'assemble'` ?

Reply:
The result of `string.split` is an array, that's why I used the include function.